### ISSUE TITLE
stan_betareg print/summary link.phi is incorrect

### ISSUE THREAD

**Issue by imadmali** [stan-dev/rstanarm#210]
#### Summary:
Print/summary method for stan_betareg is incorrect when `phi` is modeled as a scalar.

#### Description:
When `phi` is modeled as a scalar and `link.phi = NULL` the print/summary of stan_betareg says that `link.phi = log` but it should state `link.phi = identity`. The Stan code is actually modeling with an identity link. I think this can be fixed in the `stan_betareg.R` file after the model fitting takes place.

#### Reproducible Steps:
```
N <- 200
x <- rnorm(N, 2, 1)
z <- rnorm(N, 2, 1)
mu <- binomial(link = "logit")$linkinv(1 + 0.2*x)
phi <- exp(1.5 + 0.4*z)
y <- rbeta(N, mu * phi, (1 - mu) * phi)
hist(y, col = "dark grey", border = FALSE, xlim = c(0,1))
fake_dat <- data.frame(y, x, z)

fit <- stan_betareg(y ~ x, data = fake_dat, 
                    link = "logit", link.phi = NULL, 
                    algorithm = "sampling", iter = 100)
print(fit, digits = 2)
```

#### RStanARM Version:
2.15.4

#### R Version:
3.4.0

**Comment by jgabry** [comment]
I guess either the print method isn't pulling the info from the right place
or the right place has the wrong info. Did you find where this is
happening? If not I can take care of it.

**Comment by imadmali** [comment]
I have a fix on a local branch that I'll push shortly.

**Comment by jgabry** [comment]
Thanks

**Comment by jgabry** [comment]
closed with #214
